Src/BootCamp.Chapter/Demo1.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    static class Demo1
+    {
+        public static void Demo()
+        {
+            var Manager = new CredentialsManager(""); 
+            
+            Console.WriteLine("Would you like to register or login");
+
+            Console.WriteLine("Choose R for register");
+
+            Console.WriteLine("Choose L for login");
+
+            var choice = Console.ReadLine();
+
+            switch (Enum.Parse(typeof(Menu), choice))
+
+            {
+                case Menu.MenuChoices.R:
```

Comment:
Imagine you read this code withotu having no idea what it does.
What does ``Menu.MenuChoises.R`` mean? It's a horrible naming pattern. Rename MenuChoices to just Choises, because now it's redundant. Rename R to Register, L to Login.

Reply:
changed